strangerstudios/paid-memberships-pro
# Membership Level Block - with Individual Level Parts

### All Submissions:






### Changes proposed in this Pull Request:

Allows you to add a Membership Level Block with separate level parts including Level name, description, price, expiration text and checkout button. 

### How to test the changes in this Pull Request:

1. Edit a page and add the Single Level Block
2. Select a Level that the block should display


### Other information:




### Changelog entry

New Feature: Implemented a new Membership Level block
